README.md

```diff
@@ -33,6 +33,7 @@ To set up the sample application:
 <tr><td><code>COMMERCE_CLIENT_REALM_ID</code></td><td>A unique four-character ID (for example, bblx).</td></tr>
 <tr><td><code>COMMERCE_CLIENT_INSTANCE_ID</code></td><td>Instance ID within a realm (for example, 015).</td></tr>
 <tr><td><code>COMMERCE_CLIENT_SHORT_CODE</code></td><td>Region-specific merchant identifier (for example, staging-001).</td></tr>
+<tr><td><code>COMMERCE_SESSION_SECRET</code></td><td>Some unique identifier for session management (for example, thisisasecretkey).</td></tr>
```

Comment:
Are there any guidelines for our customers as to what this should be or how it should be generated? Any security consequences?

Reply:
It just needs to be unique per application. It should not be regenerated everytime the app runs, because that would invalidate any previous sessions. At the same time, each customer should have a different secret, if not then sessions could bleed between ecommerce sites. See: https://martinfowler.com/articles/session-secret.html

Reply:
Thanks for the info. Can we make this part of the documentation?

Reply:
sure, I will add and add Paul to review.

Reply:
this is in this PR, waiting for Paul to review. https://github.com/SalesforceCommerceCloud/sfcc-sample-apps/pull/148